src/NServiceBus.AcceptanceTests/Timeouts/When_endpoint_uses_outdated_persistence_but_acknowledges.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.AcceptanceTests.Timeouts
+{
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Timeout.Core;
+    using NUnit.Framework;
+
+    public class When_endpoint_uses_outdated_persistence_but_acknowledges
+    {
+        [Test]
+        public void Endpoint_should_start()
+        {
+            var context = new Context();
+
+            Scenario.Define(context)
+                .WithEndpoint<Endpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.IsTrue(context.EndpointsStarted);
+        }
+
+        public class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>(config =>
+                {
+                    config.SuppressOutdatedTimeoutPersistenceWarning();
```

Comment:
Do we need a fullblown code api for this, since this will be a "dropin" replacement should we consider just a plan appsetting instead? `NServiceBus/Xyz`